### ISSUE TITLE
No labels check is set to check GO terms by default

### ISSUE THREAD

**Issue by dosumis** [INCATools/ontology-development-kit#43]
Needs to be reset by hand.  Would be good if this could be set by starter kit, or put into instructions somewhere.

**Comment by dosumis** [comment]
Offending SPARQL:         FILTER ( regex(str(?cls), "^http://purl.obolibrary.org/obo/GO_"))

**Comment by cmungall** [comment]
Many of these will end up in ROBOT, I will refactor this part of OSK once https://github.com/ontodev/robot/pull/267 is merged

**Comment by matentzn** [comment]
Can this be closed?

**Comment by dosumis** [comment]
This check is now built into ROBOT: 

https://github.com/ontodev/robot/pull/267

... but it seems to have problems with imports.

https://github.com/ontodev/robot/issues/391

For now I could change the filter here:  https://github.com/INCATools/ontology-development-kit/blob/making_sparql_checks_generic/template/src/sparql/nolabels-violation.sparql#L11

FILTER ( regex(str(?cls), "^http://purl.obolibrary.org/obo/GO_"))
-> 
FILTER ( regex(str(?cls), "^http://purl.obolibrary.org/obo/"))

But this may make it too generic.

**Comment by matentzn** [comment]
This is now completely folded into ROBOT report. Any fixes should happen at that level.

**Comment by matentzn** [comment]
(I removed the nolabels violation altogether from ODK now)
